sdk/python/feast/registry_server.py

```diff
@@ -24,31 +35,52 @@ def __init__(self, registry: BaseRegistry) -> None:
         self.proxied_registry = registry
 
     def ApplyEntity(self, request: RegistryServer_pb2.ApplyEntityRequest, context):
-        self.proxied_registry.apply_entity(
-            entity=Entity.from_proto(request.entity),
-            project=request.project,
-            commit=request.commit,
+        assert_permissions(
+            resource=self.proxied_registry.apply_entity(
```

Comment:
Is this correct? I don't think `apply_entity` returns anything.

Reply:
@redhatHameed I think the code should change from:
```py
    def ApplyEntity(self, request: RegistryServer_pb2.ApplyEntityRequest, context):
        assert_permissions(
            resource=self.proxied_registry.apply_entity(
                entity=Entity.from_proto(request.entity),
                project=request.project,
                commit=request.commit,
            ),
            actions=CRUD,
        )
        return Empty()
```
to:
```py
        self.proxied_registry.apply_entity(
            entity=assert_permissions(
                resource=Entity.from_proto(request.entity),
                actions=CRUD,
            ),
            project=request.project,
            commit=request.commit,
        )
```

And the same for all the `apply_XYZ` methods. Can you pls take this? (and also verify why UTs did not fail)

Reply:
updated, accordingly. Thanks